yeonjae/baekjoon/BFS_DFS/2178_미로탐색.js

```diff
@@ -0,0 +1,55 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  const [n, m] = input.shift().split(" ").map(Number);
+  const [goalY, goalX] = [n - 1, m - 1];
+  const graph = input.map((el) => el.split("").map(Number));
```

Comment:
```suggestion
  const [n, m] = input[0].split(" ").map(Number);
  const graph = input.slice(1).map((el) => el.split("").map(Number));
```
- 입력값 받을 때 필요하면 shift써도 됩니다. 안쓰는 방향이 가장 베스트지만 어쩔 수 없는 케이스는 쓰셔도 무방합니다
- 위와 같은 예제로도 입력값을 받으실 수 있습니다.